app/views/decidim/vocdoni/admin/election_calendar/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<% add_decidim_page_title(t(".title")) %>
```

Comment:
Let' add a break-line (as it was before) so it easy to have the meta and body blocks visually separated.
```suggestion
<% add_decidim_page_title(t(".title")) %>

```